servicetalk-loadbalancer/src/main/java/io/servicetalk/loadbalancer/RoundRobinLoadBalancer.java

```diff
@@ -263,6 +281,17 @@ public void onNext(final Collection<? extends ServiceDiscovererEvent<ResolvedAdd
                 }
             }
 
+            @Nonnull
```

Comment:
`@Nonnull` is not necessary as it's added automatically by `@ElementsAreNonnullByDefault`

Reply:
Autogenerated after extract method...